id/index.html

```diff
@@ -0,0 +1,371 @@
+          </h2>
+
+          <p>
+            Itu adalah asumsi lampau yang salah. Sejak saat itu, banyak tim pengembangan 
+            <strong>belajar dengan susah payah</strong> bahwa merawat beberapa versi situs web dan menjaga pengkinian isinya <strong>menghabiskan lebih banyak biaya</strong> ketimbang membuat satu situs web saja yang dapat diakses secara penuh.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Penyandang disabilitas tidak menggunakan web</h2>
+
+          <p>
+            Salah. Faktanya, pada beberapa kasus internet adalah
+            <strong>satu-satunya cara</strong> bagi penyandang disabilitas untuk berkomunikasi dengan dunia. Saat "orang normal" menggunakan perasaan alami mereka seperti penglihatan dan pendengaran,
+            <strong>penyandang disabilitas harus bertopang pada teknologi</strong>.
+          </p>
+
+          <a href="https://www.w3.org/WAI/people-use-web/" target="_blank"
+            >Lebih Detail <span class="visually-hidden">tentang Penyandang disabilitas tidak menggunakan web</span></a
+          >
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Tuna netra tidak menonton film</h2>
+
+          <p>
+            Mereka pastinya menonton. Dengan bantuan 
+            <strong>deskripsi audio</strong> mereka dapat menonton dan mendengar konten media apa pun seperti halnya semua orang lain.
+          </p>
+
+          <a href="https://www.w3.org/WAI/media/av/description/" target="_blank"
+            >Lebih Detail <span class="visually-hidden">tentang Tuna netra tidak menonton film</span></a
+          >
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Aksesibilitas hanya dapat diuji oleh penyandang disabilitas</h2>
+
+          <p>
+            Ya, biasanya penyandang disabilitas adalah penguji terbaik - mereka menggunakan teknologi pembantu secara penuh. Meskipun demikian, 
+            <strong>setiap orang dapat belajar</strong> cara menguji aksesibilitas situs web.
+          </p>
+
+          <a href="https://www.w3.org/WAI/test-evaluate/" target="_blank"
+            >Lebih Detail
+            <span class="visually-hidden">tentang Aksesibilitas hanya dapat diuji oleh penyandang disabilitas</span></a
+          >
+        </section>
+
+        <p>Tunjukkan situs web ini pada atasan Anda.</p>
+
+        <div class="donation-container">
+          <script type="text/javascript" src="https://storage.ko-fi.com/cdn/widget/Widget_2.js"></script>
+          <script type="text/javascript">
+            kofiwidget2.init("Say thanks!", "#F14C4C", "B0B77IR22");
+            kofiwidget2.draw();
+          </script>
+        </div>
+      </div>
+    </main>
+
+    <footer>
+      <div class="content">
+        <p>
+          © 2024
+          <a target="_blank" href="https://www.sergeikriger.com/">Sergei Kriger</a>. All rights reserved.
+        </p>
```

Comment:
@franzaurus could you please add some info about you (link to your site, github profile or anything you want) here in the footer like in other translations.